lib/mongo/cluster/topology/load_balanced.rb

```diff
@@ -0,0 +1,126 @@
+# frozen_string_literal: true
+# encoding: utf-8
+
+# Copyright (C) 2021 MongoDB Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module Mongo
+  class Cluster
+    module Topology
+
+      # Defines behavior for when a cluster is in load-balanced topology.
+      class LoadBalanced < Base
+
+        # The display name for the topology.
+        NAME = 'LoadBalanced'.freeze
+
+        # Get the display name.
+        #
+        # @example Get the display name.
```

Comment:
I would rather remove this example while usage of this method is rather clear. But if we want to keep it, let's change to `LoadBalanced.display_name`.

Reply:
I removed all of the examples.